### ISSUE TITLE
potential Facebook login issue to investigate

### ISSUE THREAD

**Issue by davidmooreppf** [opengovernment/askthem-legacy#410]
Via @reville, investigate some pages' FB logins links, going to ::

https://www.askthem.io/users/auth/facebook

... goes for him to :: 

https://www.askthem.io/users/sign_up#_=_

... can't yet replicate incognito in Chrome, but will try.

**Comment by walter** [comment]
So here's the scenario that recreates this:
- Do not have an account on askthem.io with the same email address as your Facebook account's email address (i.e. you haven't registered previously on askthem.io, for most users)
- logout of askthem.io if you are already logged into askthem.io
- login to Facebook if you aren't already
- click "login with Facebook" from askthem.io site

I looked up @reville's account on askthem.io. It doesn't appear to be linked to a Facebook account and therefore would fall into this scenario.

Basically this is the registration process for people that want to use their Facebook details. It seems to be working as designed. I would say it could use some more explanatory text saying "Now that you have logged in from Facebook, we need a few more details" or something like that.
